apps/codelab/src/app/codelabs/angular/component-tree/component-tree.component.html

```diff
@@ -86,7 +86,7 @@ <h2 i18n>
       So far we have only one component, but as your app grows it will form a
       tree of components
     </h2>
-    <img src="assets/images/tree.png" alt="tree" height="400" />
+    <slides-tree-hierarchy-svg></slides-tree-hierarchy-svg>
```

Comment:
Can you specify the size for this component, so that it scales based on the device?

Reply:
![image](https://user-images.githubusercontent.com/40883170/51519093-f3429000-1e30-11e9-9b99-d3b8649e4e10.png)
![image](https://user-images.githubusercontent.com/40883170/51519108-ffc6e880-1e30-11e9-8eba-a3e6f09b3b0b.png)